HUDKit/Classes/Progress/HUDProgressViewController.swift

```diff
@@ -67,13 +67,16 @@ public class HUDProgressViewController: UIViewController {
         }
     }
 
+    public var blurEffect: UIBlurEffect?
+
     // MARK: - Initializer
 
-    public init(customView: UIView?, status: String?) {
+    public init(customView: UIView?, status: String?, blurEffect: UIBlurEffect? = nil) {
         super.init(nibName: "HUDProgressViewController", bundle: NSBundle(forClass: HUDProgressViewController.self))
 
         self.customView = customView
         self.status = status
+        self.blurEffect = blurEffect ?? UIBlurEffect(style: .Dark)
```

Comment:
This default value `UIBlurEffect(style: .Dark)` can be omitted. The value appear twice at here and L122.